cmodelres/CMakeLists.txt

```diff
@@ -129,3 +154,15 @@ if(SEA_EXE)
         COMMAND_EXPAND_LISTS
     )
 endif()
+
+
```

Comment:
We should add something like this for KLEE's options:

```
set(KLEE_ARGS "--simplify-sym-indices --write-cvcs --write-cov --output-module --max-memory=1000 --disable-inlining --optimize --use-forked-solver --use-cex-cache --libc=uclibc --posix-runtime --external-calls=all --only-output-states-covering-new --max-sym-array-size=4096 --max-instruction-time=30s --max-time=60min --watchdog --max-memory-inhibit=false --max-static-fork-pct=1 --max-static-solve-pct=1 --max-static-cpfork-pct=1 --switch-type=internal --search=random-path --search=nurs:covnew --use-batching-search --batch-instructions=10000 --silent-klee-assume --max-forks=512")
```

Reply:
Are these the standard args for klee? I'm unfamiliar with klee except for having set it up to test our integration.

If there's some args we don't want fixed all the time, we can take them as optional arguments, like i do with Seahorn.

Reply:
Yeah, they're mostly standard. We've been playing with the `--max-forks` a bit and we might want to make that optional in the future. I'd suggest we keep it fixed for now and add that later.